daprdocs/content/en/reference/components-reference/supported-secret-stores/hashicorp-vault.md

```diff
@@ -79,6 +79,37 @@ For Kubernetes, you can use the Helm Chart: <https://github.com/hashicorp/vault-
 {{% /codetab %}}
 
 {{< /tabs >}}
+
+
+## Multiple Key-Values per Secret
+
+HashiCorp Vault supports multiple key-values in a secret. While this behavior is ultimately dependent on the underlying [secret engine](https://www.vaultproject.io/docs/secrets#secrets-engines) configured by `enginePath`, it is something that might change the way you store and retrieve keys from Vault. For instance, multiple key-values in a secret is the behavior exposed the `secret` engine, the default engined configured by the `enginePath` field.
```

Comment:
```suggestion
HashiCorp Vault supports multiple key-values in a secret. While this behavior is ultimately dependent on the underlying [secret engine](https://www.vaultproject.io/docs/secrets#secrets-engines) configured by `enginePath`, it may change the way you store and retrieve keys from Vault. For instance, multiple key-values in a secret is the behavior exposed in the `secret` engine, the default engine configured by the `enginePath` field.
```